<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login and register system</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/keys.png</normaloff>:/images/keys.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {

border-image: url(:/images/bg.png);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="topPanel" native="true">
    <property name="geometry">
     <rect>
      <x>-80</x>
      <y>0</y>
      <width>1171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#topPanel {

border-image: url(:/images/header.jpg);

}</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>551</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#label {
color :brown;
}</string>
        </property>
        <property name="text">
         <string>Login and registration system</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="loginIcon" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>98</width>
       <height>79</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#loginIcon {

border-image: url(:/images/login.jpg);

}</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="registerForm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>391</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#registerForm {

background:rgb(255, 170, 0);
border-radius:8px;

}</string>
    </property>
    <property name="title">
     <string>Please register</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>UserName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="UserName">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Password">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Email">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Phone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Phone">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="registerBtn">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#registerBtn {

color:white;
background-color:#27a9e3;
border-width:0px;
border-radius:4px;

}

#registerBtn:hover {

background-color:#66c011;

}</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="loginForm">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#loginForm {

background:rgb(255, 170, 0);
border-radius:8px;

}</string>
    </property>
    <property name="title">
     <string>Please login</string>
    </property>
    <widget class="QPushButton" name="loginBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#loginBtn {

color:white;
background-color:rgb(0, 170, 0);
border-width:0px;
border-radius:4px;

}

#loginBtn:hover {

background-color:rgb(85, 255, 255);

}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>371</width>
       <height>66</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>UserName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UserNameLogin">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>371</width>
       <height>66</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswordLogin">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
